### PR TITLE
Ele new energies from mini aod

### PR DESCRIPTION
**Pull request by pmeridian**
=> robust regression for miniAOD (from refinedSC not from moustache for electrons)
=> energy from associated photon

Additionals improvements also to the alcaSkimming.py 
To compile need to merge with cmssw pmeridian:regression_74X

### CODE REVIEW

**Comment by shervin86** [comment]
have this been checked on non-MINIAOD?

**Comment by pmeridian** [comment]
Changes should affect only the miniaod ntuple configurarion of the alcaSkimming (new energies module is enabled only in the sequence for miniaod). If you have a specific configuration to test send a command or files to be tried

**Comment by shervin86** [comment]
one compilation error fixed in branch eleNewEnergiesFromMiniAOD

**Comment by shervin86** [comment]
a test can be run from EcalAlCaRecoProducers/ with the command:
cmsRun python/alcaSkimming.py type=ALCARERECO doTree=1 doTreeOnly=1 pdfSyst=0 jsonFile=/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions15/13TeV/Cert_246908-260627_13TeV_PromptReco_Collisions15_25ns_JSON_Silver.txt isCrab=1 skim= tagFile=config/reRecoTags/Cal_Nov2015_ICcomb_v8.py isPrivate=0 files=root://eoscms.cern.ch//eos/cms/store/group/dpg_ecal/alca_ecalcalib/ecalelf/alcarereco/13TeV/Cal_Nov2015_ICcomb_v8/DoubleEG-ZSkim-Run2015D-Prompt-v4-25nsReco/258159-260737/EcalRecalElectron_1_1_CQO.root

**Comment by shervin86** [comment]
setup_git.sh updated!

**Comment by shervin86** [comment]
a new branch ECALELF/eleNewEnergiesFromMiniAOD  is being used for testing the merge, Giuseppe will try to produce ntuples and debug

File: EcalAlCaRecoProducers/python/alcaSkimming.py [hunk 1]
```diff
@@ -148,7 +154,7 @@
 process.load('Configuration.StandardSequences.GeometryDB_cff')
 process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
 process.load('Configuration.EventContent.EventContent_cff')
-process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```
Comment (shervin86) [inline EcalAlCaRecoProducers/python/alcaSkimming.py]: why this change?

**Comment by shervin86** [comment]
@gfasanel

File: EcalAlCaRecoProducers/python/alcaSkimming.py (hunk 1 above)
Comment (pmeridian) [inline EcalAlCaRecoProducers/python/alcaSkimming.py]: well part of the modification to get auto:run2_mc working, slipped in but can be reverted back to v2

**Comment by shervin86** [comment]
in the new branch "mergeattempt" the new system works for miniAOD but it's failing for ALCARECO

**Comment by shervin86** [comment]
Everything included in 80X-devel-new
